doc/misc.asciidoc

```diff
@@ -356,3 +356,54 @@ let err, val = trying({
 The error is the left value, the correct one the right (mnemonic: “right” also means “correct”). This allows to deal with error in the same way as Go does for instance.
 
 The decorators `nullify` and `raising` are the duals of `option` or `result`
+
+=== Assertions (`gololang.Assertions`)
+
+It allows to declare (and test) an expected boolean condition in a program.
+
+==== `assert`
+
+Ensure that a boolean expression is true at runtime:
+
+[source,golo]
+----
+assert(-> 5: equals(5))
+
+# If you don't provide callback to assert(), a Runtime exception is thrown if the predicate is false
```

Comment:
Why a `RuntimeException` and not a classic [AssertionError](https://docs.oracle.com/javase/7/docs/api/java/lang/AssertionError.html)?